logging/loggen/jsonloggen.sh

```diff
@@ -0,0 +1,4 @@
+IMAGE=docker.io/mffiedler/ocp-logtest:latest
```

Comment:
@anpingli, it's prefer we don't use images from personal account.  Perhaps we can ask Mike to move them to the QE account

Reply:
@pruan-rht  Thanks.